<launch>

    <node name="turtlesim_motion_pose_2" pkg="formation" type="try_path_2.py" output="screen"/> 
        <param name="turtle" type="string" value="turtle2" />
        <param name="offset" type="double" value="-1.0" />
        <param name="other_turtle1" type="string" value="turtle1" />
        <param name="other_turtle2" type="string" value="turtle3" />
        <param name="other_turtle1_ns" type="string" value="first" />
        <param name="other_turtle2_ns" type="string" value="third" />

</launch>